raml-maven-plugin/src/main/java/uk/gov/justice/raml/maven/lintchecker/RamlProvider.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.justice.raml.maven.lintchecker;
+
+import uk.gov.justice.raml.io.files.parser.RamlFileParser;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.util.Collection;
+
+import com.google.common.annotations.VisibleForTesting;
+import org.raml.model.Raml;
+
+public class RamlProvider {
```

Comment:
Again, why do we need this extra layer? It seems like a lot of extra code, just to wrap a single call to ramlOf. Why not just inject the RamlFileParser instead?